app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,9 @@
         <%= "Logged in as #{current_user.name}" %>
         <%= link_to "My Profile", "/profile"%>
         <%= link_to "Logout", "/logout"%>
+          <% if current_user = "MerchantEmployee" %>
```

Comment:
I would like to review this line.  A current user will the user resource or the session[:user_id].  I think this needs to be something like .role or add the role method we wrote earlier into the helper methods to be able to call it in the view.